app/controllers/organizations/pets_controller.rb

```diff
@@ -1,4 +1,6 @@
 class Organizations::PetsController < Organizations::BaseController
+  include Tabbable
```

Comment:
Yes I realise now this is probably the only controller that will be using those tabs specifically. So, you could move this back into the controller. I thought you pulled it from another controller, but it's the same one.

Reply:
I was a little confused, but then I thought there must be something I'm not seeing. In the future, I'll definitely ask :)